pkg/kubelet/dockertools/manager.go

```diff
@@ -486,6 +487,15 @@ func makeEnvList(envs []kubecontainer.EnvVar) (result []string) {
 	return
 }
 
+func makeEnvMap(envs []kubecontainer.EnvVar) map[string]string {
```

Comment:
Why'd you put this minor helper function up here when its only use is 200 lines below?